src/main/resources/anniv5Resources/localization/eng/upgradespack/Cardstrings.json

```
@@ -0,0 +1,42 @@
+{
+  "${ModID}:QuickFix": {
+  "NAME": "Quick Fix",
+  "DESCRIPTION": "Gain !B! Block. NL Upgrade a random card in your hand."
+  },
+  "${ModID}:RedHotHammer": {
+    "NAME": "Red-Hot Hammer",
+    "DESCRIPTION": "Deal !D! damage. NL Can be Upgraded any number of times."
+  },
+  "${ModID}:HeartOfTheForge": {
+    "NAME": "Heart of the Forge",
+    "DESCRIPTION": "{@@}Retain. NL Gain{!M!|>1= !M!} [E]. NL Can be Upgraded any number of times. NL Exhaust."
+  },
+  "${ModID}:SharpeningSheath": {
+    "NAME": "Sharpening Sheath",
+    "DESCRIPTION": "Draw !M! cards and Upgrade them."
+  },
+  "${ModID}:CarelessSwing": {
+    "NAME": "Careless Swing",
+    "DESCRIPTION": "Deal !D! damage. NL Exhaust a random *Unupgraded card in your hand."
+  },
+  "${ModID}:SetUpShop": {
+    "NAME": "Set Up Shop",
+    "DESCRIPTION": "Upgrade !M! cards from your draw or discard pile and put them on top of your draw pile."
+  },
+  "${ModID}:MastersStrike": {
+    "NAME": "Master's Strike",
+    "DESCRIPTION": "Deal !D! damage. NL Deal !M! additional damage for each one of your Upgrades."
```

```suggestion
    "DESCRIPTION": "Deal !D! damage. NL Deal !M! additional damage for each Upgraded card."
```
If not too long I'd even add in any pile to the end

---

I worded it like that because it's meant to count cards upgraded multiple times as multiple upgrades.

---

Ahh, I see where you're coming from. I don't think this is very clear though
```suggestion
    "DESCRIPTION": "Deal !D! damage. NL Deal !M! additional damage for each upgrade on all your cards."
```
Maybe?